docs/BindingsOverview.md

```diff
@@ -168,3 +169,80 @@ e.g.
 ```sql
 UPDATE [Products].[az_func].[Leases_<FunctionId>_<TableId>] SET _az_func_AttemptCount = 0 WHERE ProductId = 123
 ```
+
+#### Lease Tables clean up
+
+Before clean up, please see [Leases table](./TriggerBinding.md#az_funcleases_) documentation for how to get the correct Leases table.
+
+- Clean up all the tables not accessed more than N number of days, Please initialize N according to your use case:
+
+```sql
+DECLARE @table_name NVARCHAR(MAX);
+DECLARE @UserFunctionId char(16);
```

Comment:
Please also add a comment explaining what the user should set this value to and how to get it so the script itself has context about what it's doing.

```suggestion
DECLARE @UserFunctionId char(16) = <Insert function ID here>`
```